util/chplenv/printchplenv.py

```diff
@@ -176,8 +178,10 @@ def compute_all_values():
     ENV_VALS['CHPL_LAUNCHER'] = chpl_launcher.get()
     ENV_VALS['CHPL_TIMERS'] = chpl_timers.get()
     ENV_VALS['CHPL_UNWIND'] = chpl_unwind.get()
+    ENV_VALS['CHPL_HOST_MEM'] = chpl_mem.get('host')
     ENV_VALS['CHPL_MEM'] = chpl_mem.get('target')
-    ENV_VALS['  CHPL_JEMALLOC'] = chpl_jemalloc.get()
+    ENV_VALS['  CHPL_HOST_JEMALLOC'] = chpl_jemalloc.get('host')
```

Comment:
Today the printed order is:

```
CHPL_HOST_MEM: jemalloc *
CHPL_TARGET_MEM: jemalloc
CHPL_MEM: jemalloc
  CHPL_HOST_JEMALLOC: bundled
  CHPL_TARGET_JEMALLOC: bundled
```

And I wonder if this would be slightly cleaner as:

```
CHPL_MEM: jemalloc
CHPL_HOST_MEM: jemalloc *
  CHPL_HOST_JEMALLOC: bundled
CHPL_TARGET_MEM: jemalloc
  CHPL_TARGET_JEMALLOC: bundled
```

Reply:
How do you like 
```
CHPL_HOST_MEM: jemalloc *
  CHPL_HOST_JEMALLOC: bundled
CHPL_MEM: jemalloc
CHPL_TARGET_MEM: jemalloc
  CHPL_TARGET_JEMALLOC: bundled
```
in order to keep `CHPL_MEM` and `CHPL_TARGET_MEM` next to each other?